pyanaconda/ui/gui/spokes/root_password.py

```diff
@@ -118,9 +119,11 @@ def initialize(self):
         self.checker.add_check(self._validity_check)
         self.checker.add_check(self._ascii_check)
 
-        # the password entries should be sensitive on first entry to the spoke
-        self.password_entry.set_sensitive(True)
-        self.password_confirmation_entry.set_sensitive(True)
+        # the password entries should be sensitive only if root is not locked from kickstart
+        allow_changes = self._users_module.CanChangeRootPassword or self.checker.policy.changesok
+        # or not self._users_module.IsRootAccountLocked
```

Comment:
I think this is a leftover commented out ?

Reply:
Yes, sorry, fixed.